pokeapp/src/components/Sidebar.js

```diff
@@ -1,22 +1,67 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import PokeDollarIcon from "../assets/pokeDollar.svg";
 import {
   typeIconMapper,
   abilityMapper,
   gameMapper,
   weaknessIconMapper,
 } from "../utilities/mappers";
-import { useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
 import { buyPokemon, sellPokemon } from "../redux/pokemonSlice";
+import { Typeahead } from "react-bootstrap-typeahead";
+import "react-bootstrap-typeahead/css/Typeahead.css";
+
+const sortAtoZ = (list) => {
+  const sortedAtoZ = list.sort((name1, name2) => {
+    if (name1 < name2) return -1;
+    if (name1 > name2) return 1;
+    return 0;
+  });
+  return sortedAtoZ;
```

Comment:
nice sorting method

Reply:
I have to agree with team captain. Clean!